### DISCUSSION TITLE
How to Display Reply Message From the Admin on the My Tickets >> Check tickets pages User ?

### DISCUSSION THREAD

**Discussion by miaherlianarohmat** [ladybirdweb/faveo-helpdesk#8217] [Q&A] [answered]
I have successfully configured email on faveo website. When I make a ticket from the user, it is successfully sent to the admin's Inbox and when the admin replies the message is successfully sent to the user's email along with the contents of the reply. But why does the My Tickets >> Check Ticket page contain no replies from the admin?
I'm a beginner in the world of programming, please help everyone. Thank you very much :)

![test](https://github.com/ladybirdweb/faveo-helpdesk/assets/83536344/ade74ce7-38ac-4fc5-b3c3-5a82e3313644)

**Comment by bhanu2217** [comment, marked as answer]
The issue will get resolved with this P.R https://github.com/ladybirdweb/faveo-helpdesk/pull/8207
